packages/core/src/lib/services/wallet-modules/wallet-modules.service.ts

```diff
@@ -137,16 +135,14 @@ export class WalletModules {
     if (!accounts.length) {
       return {
         accounts: [],
-        contract: null,
         selectedWalletId: null,
         recentlySignedInWallets: recentlySignedInWallets || [],
-        contracts: null,
+        contracts: [],
```

Comment:
I think it would be better with an empty array to keep consistency
```
accounts: [],
selectedWalletId: null,
recentlySignedInWallets: recentlySignedInWallets || [],
contracts: [],
```

Reply:
If you decide to keep it as `contracts: []` for consistency you can ignore the other comments (mark them as resolved) about changing the type from `MultiContractState` to `MultiContractState | null`.

This would mean there will always be a value for `near-wallet-selector:contracts: []` in `localStorage` whether the user is signed-in or not.